src/actions/resetPasswordActions.js

```diff
@@ -0,0 +1,47 @@
+import types from './actionTypes';
+import store from '../store';
+import axiosInstance from '../services/requestHandler';
+
+const { dispatch } = store;
+
+const { RESET_PASSWORD_LINK, RESET_PASSWORD,
+  RESET_PASSWORD_TOKEN } = types;
+
+const resetPassword = (path, password) => (
+  {
+    type: RESET_PASSWORD,
+    payload: axiosInstance().put(
+      path,
+      { user: { password, confirm: password } },
```

Comment:
I had previously stated to somehow combine reset password actions into the auth actions or user actions, same goes with the reducers

Reply:
ok